### PR TITLE
tracking-issue: smaller docker image

### PR DESCRIPTION
**Pull request by keegancsmith**
Our docker image size goes from 833mb to 13mb by not including the layer for
building and the fetched dependency source code.

### CODE REVIEW

File: internal/cmd/tracking-issue/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 
 WORKDIR /go/src/tracking-issue
 COPY main.go .
 
-RUN go get -d -v ./...
-RUN go install -v ./...
+RUN go mod init tracking-issue
+RUN CGO_ENABLED=0 go install .
+
+FROM alpine:3.11
```
Comment (tsenart) [inline internal/cmd/tracking-issue/Dockerfile]: Why 3.11 in particular?

Comment (keegancsmith) [reply internal/cmd/tracking-issue/Dockerfile]: no reason, just pinning the latest minor version.

Comment (tsenart) [reply internal/cmd/tracking-issue/Dockerfile]: Oh, I thought this version was old for some reason.

**Comment by tsenart** [comment]
@keegancsmith: Hadolint is sad in CI

```

Dockerfile:11 DL3018 Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
--
Dockerfile:13 DL3022 COPY --from should reference a previously defined FROM alias
```

**Comment by keegancsmith** [comment]
1s to download the image in the action instead of 19s :)

**Pull request merged**